src/main/java/mat/client/login/SecurityBannerModal.java

```diff
@@ -15,7 +15,14 @@
 public class SecurityBannerModal extends Modal {
     private Button acceptButton = new Button("I Accept");
     private Button declineButton = new Button("Decline");
-    private String bodyHtml = "<br/><p>This warning banner provides privacy and security notices consistent with " +
+    private String scrollStyle = "width: 600px;" +
+                                 "height: 250px;" +
+                                 "font-size: x-small;" +
+                                 "padding-right: 10px;" +
+                                 "overflow-x: auto;" +
+                                 "overflow-x: auto;" +
+                                 "text-align: justify;";
```

Comment:
this whole thing can go in MAT.css
```
.security-scrollbar {
  width: 600px;
  height: 250px;
  font-size: x-small;
  padding-right: 10px;
  overflow-x: auto;
  overflow-x: auto;
  text-align: justify;
}
```
then
`<div style="security-scrollbar">....</div>`

Reply:
It certainly can, but what we have been doing is just coding it in with styles since we are not restyling. This is really customized just for this class. Other dialogs won't use this layout.